cmd/project.go

```diff
@@ -138,27 +138,29 @@ var projectDeleteCmd = &cobra.Command{
 
 		if strings.ToLower(confirmDeletion) != "y" {
 			fmt.Printf("Aborting deletion of project: %v\n", projectName)
-		}
+			os.Exit(1)
+		} else {
```

Comment:
I would suggest that you drop the `else` since execution of that block will only proceed if condition on `if` doesn't pass (due to your addition of `os.Exit(1)`).
This would also make reviewing the PR easier since the formatting of the line would not change :)